portal-web/test/functional/com/liferay/portalweb/paths/portlet/controlmenu/SystemSettings.path

```diff
@@ -65,6 +65,11 @@
 	<td>//span[contains(@class,'custom-control-label-text') and contains(.,'${key_settingFieldName}')]/../..//input[@type='checkbox']</td>
 	<td></td>
 </tr>
+<tr>
+	<td>HYPERLINK_TEXT</td>
+	<td>//a[contains(text(),'${key_text}') and not(contains(@class,'lfr-btn-label'))]</td>
```

Comment:
I thought about adding the ID since this is a regression, so we make sure. 
![image](https://github.com/liferay-peds/liferay-portal/assets/63858122/bf1a91a7-5cf1-4497-bf68-5463e6c03171)